### DISCUSSION TITLE
Virtual Controller

### DISCUSSION THREAD

**Discussion by yjp20** [react-hook-form/react-hook-form#3163] [Ideas] [answered]
Is there an easy way of disabling the unregister hook in a Controller? I know it's easy enough to make a custom wrapper around a component, but I think for more complex use cases being able to disable unregister is a big feature.

**Comment by bluebill1049** [comment, marked as answer]
Hey Young Jin,

Have u tried shouldUnregister:false at useForm? That’s probably what u after.

Cheers
Bill

On 13 Oct 2020, at 2:36 pm, Young Jin Park <notifications@github.com<mailto:notifications@github.com>> wrote:


Is there an easy way of disabling the unregister hook in a Controller? I know it's easy enough to make a custom wrapper around a component, but I think for more complex use cases being able to disable unregister is a big feature.
